src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -91,7 +91,7 @@ namespace FourPipeBeam {
 	///// Note use of shared_ptr here is not a good pattern, not to be replicated without further discussion.
 	std::shared_ptr< AirTerminalUnit >
 	HVACFourPipeBeam::fourPipeBeamFactory(
-		int EP_UNUSED(objectType),
+		int EP_UNUSED( objectType ),
```

Comment:
Thanks for all the whitespace cleanup.  And thanks to GitHub for the `?w=1` query parameter on the URL so I can see the core changes easily :+1:.